.builds/openbsd.yml

```diff
@@ -35,9 +35,13 @@ tasks:
     cmake -G Ninja $CMAKE_EXTRA_FLAGS ..
     cmake --build . --config RelWithDebInfo
     ./bin/nvim --version
-- functionaltest: |
-    cd neovim/build
-    cmake --build . --config RelWithDebInfo --target functionaltest
```

Comment:
We don't actually know how much is impacted.  The tests always hang on a buffer_updates_spec.lua.  What if we instead skip that test and let the rest run?

```
env BUSTED_ARGS=--exclude-pattern=buffer_updates cmake --build . --config RelWithDebInfo --target functionaltest
```

Reply:
I tried individually skipping the tests one by one until I noticed the freezes were non-deterministic, at which point I gave up. I've added your code suggestion, let's see.

Reply:
Seems like there's an actual error in core/channels_spec.lua now:

```
[ RUN      ] channels can use stdio channel: 10029.50 ms ERR
test/helpers.lua:73: Expected objects to be the same.
Passed in:
(nil)
Expected:
(table: 0x0781c6982cb8) {
  [1] = 'notification'
  [2] = 'exit'
  [3] = {
    [1] = 3
    [2] = 0 } }
```

Will take a look at it tomorrow and see if I can figure out why.